qmk_cli/subcommands/setup.py

```diff
@@ -24,7 +24,7 @@ def setup(cli):
     qmk_firmware = Path(cli.args.destination)
 
     # Check on qmk_firmware, and if it doesn't exist offer to check it out.
-    if qmk_firmware.exists():
+    if qmk_firmware.exists() and os.listdir(qmk_firmware):
```

Comment:
Thinking about this I want more of a check than "some files exist" and checking for the existence of a Makefile seems like the thing most likely to exist long term.

```suggestion
    if (qmk_firmware / 'Makefile').exists():
```